module3/homework/passwordCheck/validation.cpp

```diff
@@ -1,2 +1,59 @@
 #include "validation.hpp"
-// TODO: Put implementations here
\ No newline at end of file
+
+#include<algorithm>
+
+std::string getErrorMessage(const ErrorCode error) {
+    switch (error) {
+    case Ok:
+        return "OK";
+        break;
+    case PasswordNeedsAtLeastNineCharacters:
+        return "Password Needs At Least Nine Characters";
+        break;
+    case PasswordNeedsAtLeastOneNumber:
+        return "Password Needs At Least One Number";
+        break;
+    case PasswordNeedsAtLeastOneSpecialCharacter:
+        return "Password Needs At Least One Special Character";
+        break;
+    case PasswordNeedsAtLeastOneUppercaseLetter:
+        return "Password Needs At Least One Uppercase Letter";
+        break;
+    case PasswordsDoesNotMatch:
+        return "Passwords Does Not Match";
+        break;
+    }
```

Comment:
```suggestion
        return "OK";
    case PasswordNeedsAtLeastNineCharacters:
        return "Password Needs At Least Nine Characters";
    case PasswordNeedsAtLeastOneNumber:
        return "Password Needs At Least One Number";
    case PasswordNeedsAtLeastOneSpecialCharacter:
        return "Password Needs At Least One Special Character";
    case PasswordNeedsAtLeastOneUppercaseLetter:
        return "Password Needs At Least One Uppercase Letter";
    case PasswordsDoesNotMatch:
        return "Passwords Does Not Match";
    }
```
Breaks are redundant when 'return' is used.